filebeat/harvester/config.go

```diff
@@ -6,42 +6,42 @@ import (
 	"time"
 
 	cfg "github.com/elastic/beats/filebeat/config"
-	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/filebeat/harvester/processor"
 	"github.com/elastic/beats/libbeat/common"
 )
 
 var (
 	defaultConfig = harvesterConfig{
-		BufferSize:      cfg.DefaultHarvesterBufferSize,
-		DocumentType:    cfg.DefaultDocumentType,
+		BufferSize:      16 << 10, // 16384
```

Comment:
I think the easiest solution here is to just write 16384. I don't think it 16 << 10 or any of the other variants brings a benefit.